### ISSUE TITLE
Last Modified: invalid type

### ISSUE THREAD

**Issue by Hecke29** [pdfcpu/pdfcpu#109]
Tried the latest release and also lastest master (`87915e407c5d21a710893abc788359affeaa9a21`)

Running `~/Downloads/pdfcpu-master/cmd/pdfcpu/pdfcpu validate -vv my.pdf` results in `Fatal: validateDateEntry: dict=pieceDict entry=LastModified invalid type`. 
The original goal is to merge PDFs (I got the same error there)

Potential relevant log information:
```
	<PieceInfo, <<
		<InDesign, <<
			<DocumentID, <FEFF0078006D0070002E006400690064003A00610061006300300036006200310038002D0062006400640031002D0034003400300031002D0039003800340038002D003400340036006400620038006300350039006400310030>>
			<LastModified, <FEFF0044003A00320030003100390030003800310034003100350034003000320030005A>>
			<NumberofPages, 1>
			<OriginalDocumentID, <FEFF0078006D0070002E006400690064003A00620065006200360061003800380062002D0030003100640063002D0034003200330038002D0039003300350037002D006300370036006300360037003400300062003200660030>>
			<PageUIDList, <<
				<0, 2201>
			>>>
			<PageWidthList, <<
				<0, 841.89>
			>>>
		>>>
	>>>
```

Full log can be found here: https://gist.github.com/Hecke29/58597211964a006371887c9f66226d31
I have to ask the client if I can give out the pdf file itself if needed

**Comment by hhrutter** [comment]
Thanks for reporting this! 
Very valuable input.
I'll check it out.

**Comment by hhrutter** [comment]
fixed with v0.2.5
